### DISCUSSION TITLE
starship with bash, second line of terminal will overlap over the first line

### DISCUSSION THREAD

**Discussion by lefthairynut** [starship/starship#2006] [Q&A]
Hey everybody. I don't know if it is a bug or not, starship with bash has a weird thing that second line will overlap the first line meaning that there is no new line at the end of first line but this isn't the case with with the fish shell. should I add anything to the config file or is it a bug?

**Comment by mickimnet** [comment]
@arsCom what kind of terminal are using, on what operating system and do you have a custom `starship.toml` configuration or none in both shells?

**Reply by lefthairynut** [reply]
I tested it in alacritty, gnome-terminal, konsole, rxvt-unicode and in all of them the result is the same. Im using arch linux (btw :D).
this is my configuration:
```
add_newline = false
[line_break]
disabled = true
```

**Reply by lefthairynut** [reply]
like this:
![starship_bash_bug](https://user-images.githubusercontent.com/30195179/102646498-610ca900-4179-11eb-9ecf-336b847b87f4.gif)

**Reply by mickimnet** [reply]
Does it happen with other prompts as well? I don‘t have an Arch Linux system and tried it with `bash` on macOS. Btw: what version of `bash` are you using?

**Reply by lefthairynut** [reply]
No, just starship. I use version 5.1.0(1)-release.

**Reply by mickimnet** [reply]
Sorry for the overlong delay. I'm still at odds, as it doesn't happen on my systems. What are your `$LANG` and  `$LC_*` settings?

**Reply by chipbuster** [reply]
Does it happen when the line break is enabled?

I just learned that bash doesn't properly handle multiple-codepoint emojis in prompts, so that might be the cause of it.
